packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -52,15 +52,15 @@ const AppContents = ({
             pushDataLayer({ event: 'allow_tracking' });
             setIsGtmTracking(true);
         }
-    }, [is_eu_country]);
+    }, [is_eu_country, tracking_status, pushDataLayer, is_gtm_tracking]);
```

Comment:
here as well I think - since we only want to run this effect if `is_eu_country` changes